src/ConfidentialLedger/ConfidentialLedger.Autorest/docs/Test-AzConfidentialLedgerNameAvailability.md

```diff
@@ -12,11 +12,24 @@ To check whether a resource name is available.
 
 ## SYNTAX
 
+### CheckExpanded (Default)
 ```
 Test-AzConfidentialLedgerNameAvailability [-SubscriptionId <String>] [-Name <String>] [-Type <String>]
  [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
 ```
 
+### CheckViaJsonFilePath
+```
+Test-AzConfidentialLedgerNameAvailability -JsonFilePath <String> [-SubscriptionId <String>]
+ [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### CheckViaJsonString
+```
+Test-AzConfidentialLedgerNameAvailability -JsonString <String> [-SubscriptionId <String>]
```

Comment:
What is the schema of the Json string? Is there any guidance on it?

Reply:
Typically this type of structure would need to be provided by the service team and we cannot speculate on this.